application/src/main/resources/db/migration/V2__add_data.sql

```diff
@@ -0,0 +1,71 @@
+insert into users (id, email, password)
```

Comment:
do we need to include actual ID in insert statement? I think we have with column autogenerated